packages/gafl-webapp-service/src/pages/order-complete/order-complete/route.js

```diff
@@ -36,8 +37,8 @@ export const getData = async request => {
   return {
     startTimeStringTitle,
     isSalmonLicence: permission.licenceType === mappings.LICENCE_TYPE['salmon-and-sea-trout'],
-    permissionCost: displayPermissionPrice(permission, request.i18n.getCatalog()),
-    permissionIsFree: getPermissionCost(permission) === 0,
+    permissionCost: displayPermissionPrice(permission, request.i18n.getCatalog(), transaction.payment.created_date),
```

Comment:
Do we need some tests to make sure this works if the transaction does not have a payment (eg for junior licences)?

Reply:
I was just about to ask this!